app/controllers/users_controller.rb

```diff
@@ -100,6 +113,22 @@ def authenticate_sendgrid_webhook
   def emit_lesson_notification_on_socket(lesson)
     return unless Rails.env.production?
 
+    # Notify iOS apps
+    # Swap lines below when production certificate is ready
+    #my_apn = Houston::Client.production
+    my_apn = Houston::Client.development
+    my_apn.certificate = IOS_CERTIFICATE
+    my_apn.passphrase = IOS_CERTIFICATE_PASSPHRASE
+
+    DeviceToken.all.each do |dt|
+      notification = Houston::Notification.new(device: dt.token)
+      notification.alert = lesson.title
+      notification.sound = "true"
+      notification.badge = 1
+      my_apn.push notification
+    end
```

Comment:
Unfortunately I believe this code is executed synchronously when notifying subscribers. Pretty soon we will want to make it async (in a worker).